pod install: missing resources in the app bundle (#704)
JetBrains/kotlin-multiplatform-dev-docs: topics/development/multiplatform-cocoapods-overview.md

Check the [CocoaPods documentation](https://guides.cocoapods.org/) for more information. If nothing works, and you still
encounter this error, report an issue in [YouTrack](https://youtrack.jetbrains.com/newissue?project=kt).

### Missing resources in the app bundle {initial-collapse-state="collapsed" collapsible="true"}

If your iOS app builds successfully but crashes on launch, or if resources such as custom fonts and 
images are missing from the final `.ipa` package, 

> the issue could be related to how CocoaPods integrates with your project.

Reviewer: ```suggestion
there might be an issue with how a Pod integrates with your project.
```

> The Compose Multiplatform Gradle plugin specifies where resources are located in the generated `podspec` file: 

Reviewer: ```suggestion
The Compose Multiplatform Gradle plugin specifies where resources are located in the generated `Podspec` file: 
```

Or `.podspec`? But I don't think it's `podspec`.

`spec.resources = ['build/compose/cocoapods/compose-resources']`.

> However, when you run the native `pod install` command on a clean project 

Reviewer: The "native" here is not transparent I would say — until I read the following paragraph and find out that there's also a Gradle task.

Maybe we should mention the task first, and then explain why `pod install` by itself doesn't cut it? Because I can feel interest draining in real time from people who are just looking for the correct button to press.
